lib/vp.js

```diff
@@ -529,6 +530,24 @@ class VP {
           agendaItemType = AGENDA_ITEM_TYPES.NOTA;
         }
 
+        if (!shortTitlePrefix) {
```

Comment:
Why is this check necessary? It will always be `true` as far as I can see, since you're setting `shortTitlePrefix` to `''` on line 502.

Reply:
Ah, I see, since it's in the loop. Nevermind